src/Esfa.Vacancy.Register.Api/Mappings/VacancyMapper.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System.Globalization;
+using Esfa.Vacancy.Api.Types;
+using Esfa.Vacancy.Register.Domain.Entities;
+using Esfa.Vacancy.Register.Infrastructure.Settings;
+using Address = Esfa.Vacancy.Api.Types.Address;
+
+namespace Esfa.Vacancy.Register.Api.Mappings
+{
+    public sealed class VacancyMapper
+    {
+        private readonly IProvideSettings _provideSettings;
+        private const string UnknownText = "Unknown";
+
+        public VacancyMapper(IProvideSettings provideSettings)
+        {
+            _provideSettings = provideSettings;
+        }
+
+        public ApprenticeshipVacancy MapToApprenticeshipVacancy(Domain.Entities.Vacancy vacancy)
```

Comment:
Need to keep Apprenticeship and Traineeship as separate as possible to reduce impact when any one of them changes.